src/index.html

```diff
@@ -15,8 +15,106 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav-links">
+          <li class="nav-link-item">
+            <a
+              class="nav-link is-active"
+              href="/apple"
```

Comment:
You have a space in the `href` attribute of the `<a>` tag. This can cause unexpected behavior in some browsers. The `href` attribute should not contain spaces. This issue is also present in other lines of your code. Please, correct it.